docs/rules-development-guide/master.adoc

```diff
@@ -24,81 +24,26 @@ include::topics/about-home-var.adoc[leveloffset=+3]
 // MTA Rules
 include::topics/about-rules.adoc[leveloffset=+2]
 
-// Getting Started
-include::topics/getting-started-rules.adoc[leveloffset=+1]
+== Creating YAML rules
 
-// Create Your First XML Rule
-include::topics/create-first-xml-rule.adoc[leveloffset=+2]
+Each analyzer rule is a set of instructions that are used to analyze source code and detect issues that are problematic for migration.
 
-// Review the Quickstarts
-include::topics/review-quickstarts.adoc[leveloffset=+2]
+The analyzer parses user-provided rules, applies them to applications' source code, and generates issues for matched rules. A collection of one or more rules forms a ruleset. Creating rulesets provides a way of organizing multiple rules that achieve a common goal. The analyzer CLI takes rulesets as input arguments.
 
-== Creating XML rules
+include::topics/yaml-rule-structure-syntax.adoc[leveloffset=+3]
 
-// XML Rule Structure
-include::topics/xml-rule-syntax.adoc[leveloffset=+2]
+=== Creating a basic YAML rule
 
-// Create a Basic XML Rule
-include::topics/create-basic-xml-rule.adoc[leveloffset=+2]
+This section describes how to create a basic {ProductShortName} YAML rule. This assumes that you already have {ProductShortName} installed. See the {ProductShortName} {ProductDocUserGuideURL}[_{UserCLIBookName}_] for installation instructions.
```

Comment:
Legacy question, but still...why is this section before "Creating your first rule" and do we actually nreed both? This can be definitely a post-GA question (and may have been aslked anyhow), given time constraints.

Reply:
Assumes you have MTA CLI installed, right? Really is a prerequisite, but I won't insist you label it that way now.